### PR TITLE
Update legacy_layer

### PR DESCRIPTION
**Pull request by stefanheimes**
Add the right services name.

### CODE REVIEW

**Comment by zonky2** [comment]
@stefanheimes how can I check this?

File: legacy_layer/classmap/MultiColumnWizardHelper.php [hunk 1]
```diff
@@ -62,7 +62,7 @@ public function __construct()
      */
     public function addScriptsAndStyles(Template $objTemplate)
     {
-        $serviceName = 'menatwork.multicolumnwizard-bundle.events_listener.parse_template';
+        $serviceName = 'MenAtWork\MultiColumnWizardBundle\EventListener\Contao\ParseTemplate';
```
Comment (discordier) [inline legacy_layer/classmap/MultiColumnWizardHelper.php:65]: Shouldn't this be: `$serviceName = ParseTemplate::class;` for readability purposes?

File: legacy_layer/classmap/MultiColumnWizardHelper.php [hunk 2]
```diff
@@ -90,7 +90,7 @@ public function addScriptsAndStyles(Template $objTemplate)
      */
     public function supportModalSelector($strTable)
     {
-        $serviceName = 'menatwork.multicolumnwizard-bundle.events_listener.load_data_container';
+        $serviceName = 'MenAtWork\MultiColumnWizardBundle\EventListener\Contao\LoadDataContainer';
```
Comment (discordier) [inline legacy_layer/classmap/MultiColumnWizardHelper.php:93]: Shouldn't this be: `$serviceName = LoadDataContainer::class;` for readability purposes?

File: legacy_layer/classmap/MultiColumnWizardHelper.php [hunk 3]
```diff
@@ -116,7 +116,7 @@ public function supportModalSelector($strTable)
      */
     public function changeAjaxPostActions()
     {
-        $serviceName = 'menatwork.multicolumnwizard-bundle.events_listener.initialize_system';
+        $serviceName = 'MenAtWork\MultiColumnWizardBundle\EventListener\Contao\InitializeSystem';
```
Comment (discordier) [inline legacy_layer/classmap/MultiColumnWizardHelper.php:119]: Shouldn't this be: `$serviceName = InitializeSystem::class;` for readability purposes?

File: legacy_layer/classmap/MultiColumnWizardHelper.php [hunk 4]
```diff
@@ -148,7 +148,7 @@ public function changeAjaxPostActions()
      */
     public function executePostActions($action, DataContainer $container)
     {
-        $serviceName = 'menatwork.multicolumnwizard-bundle.events_listener.execute_post_actions';
+        $serviceName = 'MenAtWork\MultiColumnWizardBundle\EventListener\Contao\ExecutePostActions';
```
Comment (discordier) [inline legacy_layer/classmap/MultiColumnWizardHelper.php:151]: Shouldn't this be: `$serviceName = ExecutePostActions::class;` for readability purposes?

**Review by zonky2 — APPROVED** [review]
hmm - könnte schön sein

**Pull request merged**